### DISCUSSION TITLE
Ability to customize colour of data displayed on map interface.

### DISCUSSION THREAD

**Discussion by aaron-manning** [directus/directus#11333] [Feature Requests]
Is anyone aware of a way to differentiate the colour/display of records displayed on the map view?

![image](https://user-images.githubusercontent.com/32957675/151637047-0f67580c-b1a2-4595-a5b0-14d4a2405fdc.png)

It would be possible to produce much more user-friendly experience if we could set a colour per record. Eg have the top two shapes (record 1, multi-poly) be blue and the bottom shape be red (record 2).

I imagine this would belong in the 'conditions' section of the map field config. Currently this is blank in 9.5.0

![image](https://user-images.githubusercontent.com/32957675/151637344-6f231175-86ab-4dde-ac04-fbf5a1a69140.png)

**Comment by benhaynes** [comment]
Whoa, _really_ interesting idea to use Conditionals here. To be clear, currently, conditionals only control the Interface options... while what you're showing is more of a Display/Layout option. Extending Conditions to support "other things" is worth looking into, but there may be some technical hurdles here.

@rijkvanzanten @Oreilles?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
